src/test/java/alinaTinca/tests/Tema5/Heroku/DynamicTableTestPageClass.java

```diff
@@ -0,0 +1,78 @@
+package alinaTinca.tests.Tema5.Heroku;
+
+import AlinaTinca.BrowserManagerClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class DynamicTableTestPageClass {
+
+    public static void main(String[] args) {
+        goToTheDynamicTableTestPage();
+        //check if the method called goToTheDynamicTableTestPage is done successfully
+        getTableData("Alina", "123");
+        //check if the method called getTableData is done successfully
+    }
+    public static void goToTheDynamicTableTestPage() {
+        //create a method called goToTheDynamicTableTestPage that goes to the link called The Dynamic Table Test Page from this URL and makes different operations there
+        ChromeDriver driver = BrowserManagerClass.createChromeDriver();
+        //create and configure the Chrome browser page settings
+        driver.get("https://testpages.herokuapp.com/styled/index.html");
+        //go to the Heroku URL
+        WebElement Link4TDynamicTable = driver.findElement(By.id("dynamictablestest"));
+        //search the 5th link
+        Link4TDynamicTable.click();
+        //click on the 5th link
+        WebElement firstParagraph = driver.findElement(By.cssSelector(".explanation"));
+        //search the first paragraph
+        System.out.println("The first paragraph has the following content: " + firstParagraph.getText());
+        //display the message from the first paragraph
+        WebElement tableSTitle = driver.findElement(By.xpath("//caption[text()='Dynamic Table']"));
+        //search the table's title
+        System.out.println("The table's title is called: " + tableSTitle.getText());
+        //display the table's title
+        WebElement table = driver.findElement(By.cssSelector("#dynamictable"));
+        //search the table
+        System.out.println("The table has the following body: " + table.getText());
+        //display the table's content
+        WebElement buttonTableData = driver.findElement(By.xpath("//summary[text()='Table Data']"));
+        //search the button called Table Data
+        buttonTableData.click();
+        //click on the button called Table Data
```

Comment:
You forgot to close the driver here and the instance will remain open. Please add the line in this method. Thanks!